bin/rucio-kronos

```diff
@@ -37,6 +38,7 @@ def get_parser():
     parser = argparse.ArgumentParser()
     parser.add_argument("--run-once", action="store_true", default=False, help='One iteration only')
     parser.add_argument("--threads", action="store", default=1, type=int, help='Concurrency control: number of threads')
+    parser.add_argument('--sleep-time', action="store", default=60, type=int, help='Seconds to sleep between two cycles')
```

Comment:
might be better to add two different sleep time parameters. One for the file and one for the dataset thread. The dataset thread can usually run with a long sleep time to aggregate some datasets before updating.